.travis.yml

```diff
@@ -0,0 +1,49 @@
+language: julia
+
+os:
+  - linux
+  - osx
+  - freebsd
+  - windows
+
+julia:
+  - 1.0
+  - 1.4
+  - nightly
+
+notifications:
+  email: false
+
+cache:
+  directories:
+    - $HOME/.julia/artifacts
+
+branches:
+  only:
+  - master
+  - gh-pages # For building documentation
+  - /^testing-.*$/ # testing branches
+  - /^v[0-9]+\.[0-9]+\.[0-9]+$/ # version tags
+
+jobs:
+  allow_failures:
+    - julia: nightly
+  include:
+    - stage: "Documentation"
+      julia: 1.3
```

Comment:
similar, it's fine to pin this at 1.3 but you may want to use `1` if you just want "the most up to date stable release"